.travis.yml

```diff
@@ -5,11 +5,15 @@ go:
 
 go_import_path: github.com/cloudflare/unsee
 
-# install jshint & eslint, it's used for linting js files (needs nodejs)
```

Comment:
You may consider using [Travis Build Stages](https://docs.travis-ci.com/user/build-stages/) to run lint tests separately from unit tests.